server/common.ts

```diff
@@ -214,52 +215,33 @@ import * as mongoose from "mongoose";
 mongoose.connect(url.resolve(config.server.mongoURL, config.server.uniqueAppID));
 export {mongoose};
 
-import {
-	ISettingMongoose, Setting
-} from "./schema";
+import { Setting } from "./schema";
 
 export async function setDefaultSettings() {
-	if (await Setting.find({ "name": "applicationOpen" }).count() === 0) {
-		await new Setting({
-			"name": "applicationOpen",
-			"value": new Date()
-		}).save();
-	}
-	if (await Setting.find({ "name": "applicationClose" }).count() === 0) {
-		await new Setting({
-			"name": "applicationClose",
-			"value": new Date()
-		}).save();
-	}
-	if (await Setting.find({ "name": "confirmationOpen" }).count() === 0) {
-		await new Setting({
-			"name": "confirmationOpen",
-			"value": new Date()
-		}).save();
-	}
-	if (await Setting.find({ "name": "confirmationClose" }).count() === 0) {
-		await new Setting({
-			"name": "confirmationClose",
-			"value": new Date()
-		}).save();
-	}
-	if (await Setting.find({ "name": "teamsEnabled" }).count() === 0) {
-		await new Setting({
-			"name": "teamsEnabled",
-			"value": true
-		}).save();
-	}
-	if (await Setting.find({ "name": "applicationBranches" }).count() === 0) {
-		await new Setting({
-			"name": "applicationBranches",
-			"value": []
-		}).save();
-	}
-	if (await Setting.find({ "name": "confirmationBranches" }).count() === 0) {
-		await new Setting({
-			"name": "confirmationBranches",
-			"value": []
-		}).save();
+	async function doesNotExist(key: string): Promise<boolean> {
+		return await Setting.find({ "name": key }).count() === 0;
+	}
+
+	if (await doesNotExist("applicationOpen")) {
+		await updateSetting("applicationOpen", new Date());
```

Comment:
Warn users about what? This function sets default settings values when the app starts up for the first time and the DB is empty, only writing settings if they do not exist.